libs/cli/README.md

```diff
@@ -1,3 +1,10 @@
 # langchain-cli
 
-This package implements the official CLI for LangGraph API.
\ No newline at end of file
+This package implements the official CLI for LangGraph API.
+
+## How to Test Locally
```

Comment:
i think this is a bit misleading to a consumer of the library, maybe let's clarify that this is for langgraph cli development?